snowex_db/upload/points.py

```diff
@@ -203,19 +203,16 @@ def submit(self):
 
             # Grab each row, convert it to dict and join it with site info
             if not df.empty:
-                # IMPORTANT: Add observations first, so we can use them in the
-                # entries
-
                 c_observations = self._add_campaign_observation(df)
+                measurement_types = self._add_measurement_types(df)
 
                 for row in df.to_dict(orient="records"):
                     row["geometry"] = WKTElement(
-                        str(row["geometry"]),
-                        srid=int(df.crs.srs.replace("EPSG:", ""))
+                        str(f"POINT ({row['longitude']} {row['latitude']})"),
+                        srid=4326,
```

Comment:
This made me drop the use of GeoDataframes, since we are converting the geometry column back to a string. The Layer data import already used this pattern.